controllers/commentsController.js

```diff
@@ -44,4 +45,19 @@ const postCommentByArticleId = (req, res, next) => {
     });
 };
 
-module.exports = { getCommentsByArticleId, postCommentByArticleId };
+const deleteCommentByCommentId = (req, res, next) => {
+  const { comment_id } = req.params;
+  removeCommentByCommentId(comment_id)
+    .then(() => {
+      res.status(204).send();
```

Comment:
When you don't want to send anything back you can use res.sendStatus(204) to only send the status code